docs/azure/includes/dotnet-all.md

```diff
@@ -163,14 +165,16 @@
 | Key Vault - Keys | NuGet [4.1.0](https://www.nuget.org/packages/Azure.Security.KeyVault.Keys/4.1.0)<br>NuGet [4.2.0-beta.3](https://www.nuget.org/packages/Azure.Security.KeyVault.Keys/4.2.0-beta.3) | [docs](/dotnet/api/overview/azure/Security.KeyVault.Keys-readme/) | GitHub [4.1.0](https://github.com/Azure/azure-sdk-for-net/tree/Azure.Security.KeyVault.Keys_4.1.0/sdk/keyvault/Azure.Security.KeyVault.Keys/)<br>GitHub [4.2.0-beta.3](https://github.com/Azure/azure-sdk-for-net/tree/Azure.Security.KeyVault.Keys_4.2.0-beta.3/sdk/keyvault/Azure.Security.KeyVault.Keys/) |
 | Key Vault - Secrets | NuGet [4.1.0](https://www.nuget.org/packages/Azure.Security.KeyVault.Secrets/4.1.0)<br>NuGet [4.2.0-beta.3](https://www.nuget.org/packages/Azure.Security.KeyVault.Secrets/4.2.0-beta.3) | [docs](/dotnet/api/overview/azure/Security.KeyVault.Secrets-readme/) | GitHub [4.1.0](https://github.com/Azure/azure-sdk-for-net/tree/Azure.Security.KeyVault.Secrets_4.1.0/sdk/keyvault/Azure.Security.KeyVault.Secrets/)<br>GitHub [4.2.0-beta.3](https://github.com/Azure/azure-sdk-for-net/tree/Azure.Security.KeyVault.Secrets_4.2.0-beta.3/sdk/keyvault/Azure.Security.KeyVault.Secrets/) |
 | Key Vault - WebKey | NuGet [3.0.5](https://www.nuget.org/packages/Microsoft.Azure.KeyVault.WebKey/3.0.5) |  | GitHub [3.0.5](https://github.com/Azure/azure-sdk-for-net/tree/Microsoft.Azure.KeyVault.WebKey_3.0.5/sdk/keyvault/Microsoft.Azure.KeyVault.WebKey/) |
+| Kinect Developer Kit | NuGet [1.0.1](https://www.nuget.org/packages/Microsoft.Azure.Kinect.BodyTracking /1.0.1) |  |  |
```

Comment:
Markdown failure is due to the extra space. That needs to be fixed on the tool which the bot uses to generate PRs.

```suggestion
| Kinect Developer Kit | NuGet [1.0.1](https://www.nuget.org/packages/Microsoft.Azure.Kinect.BodyTracking/1.0.1) |  |  |
```

Reply:
@weshaggard Hi Wes, can you please take a look at the markdown linting error @Youssef1313 mentions?

Reply:
I fixed this in the underlying datasource in https://github.com/Azure/azure-sdk/commit/0ba285c0736874067224d8a5e0b418ab296e2583 so the next time the automate runs to update this MD file it should be fixed.

Reply:
You can fix it directly here if you want to get this PR merged.

Reply:
The latest push has this updated.